networkx/generators/geometric.py

```diff
@@ -272,7 +272,7 @@ def soft_random_geometric_graph(
     --------
     Default Graph:
 
-    G = nx.soft_random_geometric_graph(50, 0.2)
+    >>> G = nx.soft_random_geometric_graph(50, 0.2)
 
     Custom Graph:
```

Comment:
Hi, just came accross this PR. If I understand correctly, the example for Custom Graph is in the Notes section, a few lines below in the source code.

However, in the [doc](https://networkx.org/documentation/latest/reference/generated/networkx.generators.geometric.soft_random_geometric_graph.html), the Notes section appears above the Examples section, which makes this example confusing. I think it would be perfect if both examples for Default Graph and Custom Graph could be placed in the same Examples section.

Reply:
Good catch. It is weird that `Examples` is at the bottom... below even `References`.

Reply:
I am (as usual) confused.  It looks to me like both the Default Graph example and the Custom Graph example appear together in an Examples section after the References section.  I see that the Notes section is above both Refernece and Examples. That is, the section order changes between the doc_string in the code and the HTML generated by sphinx using the theme.  But both examples are in the same examples section.

Is that different from what you are seeing @yuanx749 ?

Reply:
@dschult What I feel confusing is that there is only instruction ('Create a soft random...') under Custom Graph, but no code example. (Default Graph has example code.)

My understanding may be wrong. The code snippet in the Notes section could be used as an example for custom graph, but it appears above Examples section due to the section order changes after sphinx rendering.

Reply:
Ahhh....  I understand what you are saying now. :)
Thank you!

Reply:
Agreed that the `Examples` section of some of the geometric generators could use cleanup too - @yuanx749 would you want to open a separate PR with proposed updates there?

They can be fixed here as well, but IMO the proposed changes as of dfbead5 are good to go!

Reply:
@rossbar Yes, I think it would be better to update the examples with a separate PR. (I may open one in a couple of days if I have time).